app/templates/realdealz/game_list.html

```diff
@@ -5,29 +5,95 @@
   li {
     font-size: 18px;
   }
+
+  table {
+    table-layout: fixed;  
+  }
+
+  th.id, td.id {
+    width: 50px;
+  }  
+
+  th.name, td.name {
+    width: 400px;
+  }
+
+  th.price, td.price {
+    width: 100px;
+  }
+
+  th.developer, td.developer {
+    width: 800px;
+  }
+
+  th, td {
+  word-wrap: break-word;
+  }
+
```

Comment:
could also put these in static and call upon them from there, but this does work

Reply:
I agree I actually didn’t write that part not sure what happened there i might have merged weird or something